Flabbergasted-Flamingos/Bangazon-admin-app
# Jcaddemployee

# Description
Can add new employee to DB

## Number of Fixes
0

## Related Ticket(s)
#2 

## Problem to Solve
Describe the problem(s) this pull request solves.

## Proposed Changes
Describe the proposed changes.

## Expected Behavior
Describe the expected behavior.

## Steps to Test Solution

1. git fetch all
2. git checkout JCaddemployee
3. add a department in the DB
4.  go to URL 'http://localhost:8000/admin_app/employees/add/'
5. The form should render.  
6. If renders add an employee
7. Should redirect to employee list and you she see the new employee

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods
